<launch>
  <!-- load lasso mode params -->
  <rosparam file="$(find ar_star_ros)/params/lasso_mode.yaml" command="load"/>
  
  <!-- Launch the service server -->
  <node pkg="ar_star_ros" type="lasso_solver" name="lasso_solver" output="screen" />

  <!-- Launch the service client -->
  <node pkg="ar_star_ros" type="lasso_listener" name="lasso_listener" output="screen" />
</launch>